pkg/config/cmd/cmd.go

```diff
@@ -165,6 +165,8 @@ func (b *BulkAction) BindForAction(flags *pflag.FlagSet) {
 func (b *BulkAction) BindForOutput(flags *pflag.FlagSet) {
 	flags.StringVarP(&b.Output, "output", "o", "", "Output results as yaml or json instead of executing, or use name for succint output (resource/name).")
 	flags.BoolVar(&b.DryRun, "dry-run", false, "If true, show the result of the operation without performing it.")
+	flags.Bool("no-headers", false, "Omit table headers for default output.")
```

Comment:
PrintOptions needs to be a Struct that we call Bind on that is selective.

On Sep 14, 2016, at 1:20 PM, David Eads <notifications@github.com> wrote:

In pkg/config/cmd/cmd.go
<https://github.com/openshift/origin/pull/10777#discussion_r78793612>:

> @@ -165,6 +165,8 @@ func (b *BulkAction) BindForAction(flags *pflag.FlagSet) {
>  func (b *BulkAction) BindForOutput(flags *pflag.FlagSet) {
>  	flags.StringVarP(&b.Output, "output", "o", "", "Output results as yaml or json instead of executing, or use name for succint output (resource/name).")
>  	flags.BoolVar(&b.DryRun, "dry-run", false, "If true, show the result of the operation without performing it.")
> +	flags.Bool("no-headers", false, "Omit table headers for default output.")

blech. @fabianofranz <https://github.com/fabianofranz> this should be fixed
upstream.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/10777/files/63fd2c03f83157f74a8aca4046256be10c8b7dff..aed693d630231919dee7b15e5a0b0e7e07c182e2#r78793612>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABG_pxBKrEooWGsFOnlb2t14tmZG3dOxks5qqCzsgaJpZM4JzJrj>
.